projects/sbb-esta/angular-business/src/lib/contextmenu/contextmenu/contextmenu.component.scss

```diff
@@ -0,0 +1,87 @@
+.mask-right {
```

Comment:
I'd prefer to use namespacing for css classes to avoid conflicts for the consumer
e.g. use .sbb-contextmenu-mask-right here